apps/api/src/app/subscribers-v2/usecases/list-subscribers/list-subscribers.command.ts

```diff
@@ -7,9 +7,9 @@ export class ListSubscribersCommand extends CursorBasedPaginatedCommand {
   @IsOptional()
   orderDirection: DirectionEnum = DirectionEnum.DESC;
 
-  @IsEnum(['updatedAt', 'createdAt'])
+  @IsEnum(['updatedAt', '_id'])
   @IsOptional()
-  orderBy: 'updatedAt' | 'createdAt' = 'createdAt';
+  orderBy: 'updatedAt' | '_id' = '_id';
```

Comment:
as far as i understand sorting by `id` should be an internal implementation, API client should not be aware of it and it should sort by `createdAt`. we need to map `createdAt` to `id` internally